Dockerfile.dev

```diff
@@ -1,8 +1,7 @@
 # To run: docker run --rm -d -v /path/to/amanuensis-config.yaml:/var/www/amanuensis/amanuensis-config.yaml --name=amanuensis -p 80:80 amanuensis
 # To check running container: docker exec -it amanuensis /bin/bash
 
-FROM quay.io/cdis/python:python3.6-buster-pybase3-3.0.2
-# FROM quay.io/pcdc/python:3.6-buster
+FROM quay.io/cdis/python:python3.9-buster-master
```

Comment:
I forgot, what feature needed 3.9?

Reply:
I contacted the CTDS team, the most up to date stable python 3.9 container they have is `quay.io/cdis/python:python3.9-buster-2022.07`